.github/workflows/reusable-python-build_poetry_package.yml

```diff
@@ -11,6 +11,11 @@ on:
             description: 'Python version to use (eg: 3.10)'
             required: true
             type: string
+        lfs:
+            description: 'Boolean to indicate if Github LFS is needed'
+            required: false
+            type: boolean
+            default: false
```

Comment:
**suggestion:** just always use LFS. No need for an input parameter

it will cost a bit if there are LFS files that we are not packaging, but building package is not action that will run so often, supposedly.

And it is too easy to forget to turn the lfs option to true.

Reply:
Why not